frontend/src/old-pages/Configure/Configure.tsx

```diff
@@ -301,22 +308,26 @@ function Configure() {
   useWizardSectionChangeLog()
 
   return (
-    <Layout contentType="form">
-      <StopDialog clusterName={clusterName} />
-      <SpaceBetween direction="vertical" size="l">
-        <BreadcrumbGroup
-          items={[
-            {text: 'Clusters', href: '#clusters'},
-            // @ts-expect-error TS(2741) FIXME: Property 'href' is missing in type '{ text: string... Remove this comment to see the full error message
-            {text: editing ? 'Update' : 'Create'},
-          ]}
+    <Layout
+      contentType="form"
+      breadcrumbs={
+        <Breadcrumbs
           onClick={clearStateAndCloseWizard}
+          pages={[
+            breadcrumbItemFromSlug('clusters'),
+            breadcrumbItem(editing ? 'Update' : 'Create'),
+          ]}
```

Comment:
issue: unless you are memoizing everything properly, creating a new array at every render could cause excessive re-renders

As stated in the other comments, one solution could be:
- just pass a key to the Layout component (e.g. `clusters-update`)
- the Layout component could use `useMemo` if needed (double check if re-renders cause a rerender of the full layout)